plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestTrinoS3FileSystem.java

```diff
@@ -353,7 +356,9 @@ public void testReadForbidden()
             fs.initialize(new URI("s3n://test-bucket/"), new Configuration(false));
             fs.setS3Client(s3);
             try (FSDataInputStream inputStream = fs.open(new Path("s3n://test-bucket/test"))) {
-                inputStream.read();
+                assertThatThrownBy(() -> inputStream.read())
+                        .isInstanceOf(IOException.class)
+                        .hasMessageContaining(".*Failing getObject call with " + HTTP_FORBIDDEN + ".*");
```

Comment:
The expected message is wrong. 
```suggestion
                assertThatThrownBy(inputStream::read)
                        .isInstanceOf(IOException.class)
                        .hasMessageContaining("Failing getObject call with " + HTTP_FORBIDDEN);
```